adafruit_displayio_layout/widgets/widget.py

```diff
@@ -166,14 +172,14 @@ class Widget(displayio.Group):
 
     def __init__(
         self,
-        x=0,
-        y=0,
-        scale=1,
-        width=None,
-        height=None,
-        anchor_point=None,
-        anchored_position=None,
-    ):
+        x: int = 0,
+        y: int = 0,
+        scale: float = 1,
```

Comment:
this `scale` gets passed to displayio.Group constructor. I believe it can only use `int` not `float` for valid values.